build/Foundation/dll/Foundation.def

```diff
@@ -874,4 +874,11 @@ LIBRARY Foundation
         ; MainDispatcher
         ScheduleMainRunLoop
         ScheduleMainRunLoopAsync
-        SetupMainRunLoopTimedMultipleWaiter
\ No newline at end of file
+        SetupMainRunLoopTimedMultipleWaiter
+
+        ;UTTypeHelpers (Internal)
+        _UTTypeCreatePreferredIdentifierForTag
+        _UTTypeCreateAllIdentifiersForTag
```

Comment:
and much more offline talks ended up in just compiling it as C++ to get over the error linking issues.

---
In reply to: [95277632](https://github.com/Microsoft/WinObjC/pull/1662#discussion_r95277632) [](ancestors = 95277632,95276230,95276171)